js/common/modules/jsunity.js

```diff
@@ -47,7 +47,7 @@ function setTestFilter(filter) {
 }
 
 jsUnity.results.begin = function (total, suiteName) {
-  print('Running ' + (suiteName || 'unnamed test suite'));
+  print(Date() + ' Running ' + (suiteName || 'unnamed test suite'));
```

Comment:
Probably, explicit `suiteName !== null` should be here.

http://es5.github.io/#x8.2

Reply:
hm, I didn't want to alter current behaviour; if, we maybe should review the jsunity completely...